.buildkite/steps/release-github.sh

```diff
@@ -2,13 +2,7 @@
 
 set -euo pipefail
 
-splitter_version="$(cat version/VERSION)" 
-
-# Skip release if version already exists
```

Comment:
This check has moved to the `release-prepare.sh`. We will not run this job if the tag is already exists.